tensorflow-tools/src/main/java/org/tensorflow/tools/Shape.java

```diff
@@ -19,16 +19,29 @@
 
 import java.util.Arrays;
 
+/**
+ * The shape of a Tensor or {@link org.tensorflow.tools.ndarray.NdArray}. Defines the size along
+ * its axis. May contain an unknown size for one of the axis or may be totally unknown, in which
+ * case not even the number of axis is known.
+ */
```

Comment:
Funny, I never realized that `Shape` was missing that much documentation... thanks!

Can you please reformat it to follow [Google Java Style guide](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs)? "Defines the.." should start on a separate line, preceded by a blank line plus a `<p>` tag, like [here](https://github.com/tensorflow/java/blob/ef0b4f26254cbbfd29b39e184962706c2d348e59/tensorflow-tools/src/main/java/org/tensorflow/tools/ndarray/NdArray.java#L26). This should be applied to all other javadoc blocks as well.

Reply:
Sure! Sorry for not checking the styleguide first.